packages/jest-runner/corpus.ts

```diff
@@ -18,49 +18,57 @@ import path from "path";
 import fs from "fs";
 
 export class Corpus {
-	private readonly _inputDirectory: string;
-	private readonly _outputDirectory: string;
+	private readonly _inputsDirectory: string;
 
 	constructor(testPath: string, testStateElements: string[]) {
-		this._inputDirectory = this.buildInputDirectory(
+		this._inputsDirectory = this.buildInputsDirectory(
 			testPath,
 			testStateElements
 		);
-		this._outputDirectory = this.buildOutputDirectory(this._inputDirectory);
-		fs.mkdirSync(this._inputDirectory, { recursive: true });
-		fs.mkdirSync(this._outputDirectory, { recursive: true });
+		this.createMissingDirectoriesAndSeeds();
 	}
 
-	get inputDirectory(): string {
-		return this._inputDirectory;
+	get inputsDirectory(): string {
+		return this._inputsDirectory;
 	}
 
-	get outputDirectory(): string {
-		return this._outputDirectory;
+	inputsPaths(): [string, string][] {
+		return fs
+			.readdirSync(this._inputsDirectory)
+			.filter(
+				(entry) =>
+					!fs.lstatSync(path.join(this.inputsDirectory, entry)).isDirectory()
+			)
+			.map((file) => [file, path.join(this._inputsDirectory, file)]);
 	}
 
-	inputPaths(): [string, string][] {
-		return fs.readdirSync(this._inputDirectory).map((file) => {
-			return [file, path.join(this._inputDirectory, file)];
-		});
-	}
-
-	private buildOutputDirectory(inputDirectory: string): string {
-		return inputDirectory + path.sep;
-	}
-
-	private buildInputDirectory(
+	private buildInputsDirectory(
 		testPath: string,
 		testStateElements: string[]
 	): string {
 		const root = path.parse(testPath);
 		const testElements = testStateElements.map(
 			this.replaceSpacesWithUnderscore
 		);
-		return path.join(root.dir, root.name, ...testElements);
+		return path.join(root.dir, root.name, ...testElements) + path.sep;
 	}
 
 	private replaceSpacesWithUnderscore(s: string): string {
 		return s.replace(/ /g, "_");
 	}
+
+	private createMissingDirectoriesAndSeeds() {
+		fs.mkdirSync(this._inputsDirectory, { recursive: true });
+		// As we always want to start regression tests with an empty input, as kind
```

Comment:
Two questions:
1. Could you clarify why we need to handle empty inputs as a special case?
2. Recreating the empty input after the user has deleted it might not be the best user experience, WDYT?

Reply:
Well, wasn't our idea to invoke the fuzz test with an empty input as kind of a smoke test? 
I further discussed this feature with @oetr and we also came to the same conclusion: Recreating an empty input file feels strange. 
This functionality is now removed. Tests without seed files are now marked as skipped, which prevents Jest from throwing errors due to missing tests. WDYT?